platform/web/detect.py

```diff
@@ -34,6 +34,9 @@ def get_opts():
 
     return [
         ("initial_memory", "Initial WASM memory (in MiB)", 32),
+        # default STACK_SIZE and DEFAULT_PTHREAD_STACK_SIZE were reduced from 5 MiB and 2 Mib to 64Kib in emscripten 3.1.27
```

Comment:
```suggestion
        # Default STACK_SIZE and DEFAULT_PTHREAD_STACK_SIZE were reduced from 5 MiB and 2 MiB to 64 KiB in Emscripten 3.1.27
```
But I'm not sure we need this comment at all, or it could be simplified further:
```suggestion
        # Matches default values from before Emscripten 3.1.27. New defaults are too low for Godot.
```